src/psyclone/transformations.py

```diff
@@ -3293,24 +3277,17 @@ def apply(self, nodes, options=None):
 
         parent = node_list[0].parent
         schedule = node_list[0].root
+        start_index = parent.children.index(node_list[0])
```

Comment:
I think you can just have `node_list[0].position`?

Reply:
Done